how-to/how-to-navigate-local-logs.md

```diff
@@ -0,0 +1,32 @@
+---
+title: How to Navigate to The Extension for Visual Studio and The Classic Windows Desktop Client Logs
+description: Step by step tutorial for navigating to the Telerik Platform local Windows logs.
+type: how-to
+page_title: How to Navigate to AppBuilder Windows Clients Local Logs
+slug: navigate-local-logs
+position:
+tags: other
+teampulseid: 
+ticketid: 
+pitsid: 
+---
+                    
+## Environment
+<table>
+  <tr>
+    <td>{{site.ab-s}} ({{site.ab}}) clients</td>
+    <td>Extension for Visual Studio, classic Windows Desktop</td>	
+  </tr>
+</table>
+
+## Description
+
+This article provides the general steps for navigating to the Extension for Visual Studio and the classic Windows Desktop client logs.
+
+## Solution
+
+{{site.tab}} extension for Visual Studio and classic Windows Desktop store logs in the **Logs** folder, located in the **%localappdata%\Telerik\BlackDragon** directory.
+
```

Comment:
You can add a sentence similar to the one in the Fiddler logs article:
"This will produce a SAZ file, which you will be able to archive and share with the Telerik Platform support personnel."
http://docs.telerik.com/platform/knowledge-base/how-to/how-to-capture-fiddler-trace

Something like, "You can archive and share these logs with the Telerik Platform support personnel when you report a crash in the AppBuilder clients.". Feel free to fully re-word this.